ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog
 
+## v0.1.0.3
+- C++ compatible printing and parsing of text format. It is incompatible
```

Comment:
- In printing and parsing protocol buffers use/interpret escape sequences in a way compatible with the Protocol Buffer distribution's C/C++ escaping style, closing Issues #44 and #45.  This makes the text output incompatible with previous versions; earlier versions will misinterpret text written by this version and this version will misinterpret text written by previous versions.

- Support reading text protocol buffers with single quote characters (') for string delimiters, closing Issue #20.